config/stored_requests.go

```diff
@@ -11,8 +10,20 @@ import (
 	"github.com/golang/glog"
 )
 
+// DataType constants
+type DataType string
+
+const (
+	RequestDataType    DataType = "Request"
+	CategoryDataType   DataType = "Category"
+	VideoDataType      DataType = "Video"
+	AmpRequestDataType DataType = "Amp Request"
```

Comment:
Nitpick: AMP is an acronym. Following Go naming conventions it should  be "AMPRequestDataType".